subsys/net/lib/lwm2m/Kconfig.ipso

```diff
@@ -39,6 +39,82 @@ config LWM2M_IPSO_TEMP_SENSOR_TIMESTAMP
 	help
 	  Add a non-standard timestamp resource to each temperature object.
 
+config LWM2M_IPSO_GENERIC_SENSOR
+	bool "IPSO Generic Sensor Support"
+	help
+	  This IPSO object can be used to prototype a sensor.
+
+config LWM2M_IPSO_GENERIC_SENSOR_INSTANCE_COUNT
+	int "Maximum # of IPSO Generic Sensor object instances"
+	default 1
+	depends on LWM2M_IPSO_GENERIC_SENSOR
+	help
+	  This setting establishes the total count of IPSO Generic
+	  Sensor instances available to the LWM2M client.
+
+config LWM2M_IPSO_GENERIC_SENSOR_TIMESTAMP
+	bool "Add timestamp resources to IPSO objects"
+	default y
+	depends on LWM2M_IPSO_GENERIC_SENSOR && LWM2M_IPSO_TIMESTAMP_EXTENSIONS
+	help
+	  Add a non-standard timestamp resource to each object.
+
+config LWM2M_IPSO_GENERIC_SENSOR_IPSO_OBJECT_ID
```

Comment:
I'm a bit confused about this config, isn't Generic Sensor IPSO object supposed to have a fixed ID (3300) as it's defined in the registry?
https://raw.githubusercontent.com/OpenMobileAlliance/lwm2m-registry/prod/3300.xml